book/2-proj-mgmt-tech-lead/translate-technical-jargon.md

```diff
@@ -0,0 +1,33 @@
+# How might we simplify and translate technical jargon for a non-technical audience?
+Contributor: Shafir Ahmad, Senior AI Technical Consultant
+
+Communicating with people who are not actively involved in your project can present a challenge. Technical staff such as engineers and developers find it difficult to communicate complex technical concepts to clients, senior management and customers alike. It might even look like you’re speaking different languages.
+
+The fact is that many of these stakeholders you communicate with will understand the big picture and the high level concepts, but not the intricate engineering details. It is better to keep these details for a more technically inclined audience. 
+
+Here are some tips you can use to reach an understanding between yourself and non-technical stakeholders.
+
+## 1. Know your audience
+
+Learn about your audience. Find out what their knowledge is on the subject matter beforehand so that you can tailor the information to their needs. During early engagements, it is better to communicate at a level below what you reasonably expect them to know, while being careful not to dumb it down too much such as to cause offence. When in doubt, you could always let them know “If this is something you already know, let us know and we can skip to the next section.” Do continually observe the stakeholder’s gestures, facial expressions and body language to ensure everyone is on the same page. 
+
+## 2. Reduce or Eliminate Tech Jargon
+
+Start eliminating tech jargon and acronyms from your conversations with the stakeholders. Certain terms may not make any sense to the stakeholder, and others may hold a different meaning from the one you commonly use (e.g. NLP = Neuro Linguistic Programming vs NLP = Natural Language Processing). Although it may seem like there is no way to communicate the details without using the jargon you are used to, there are ways to mitigate this. Consider the use of metaphors, analogies and storytelling techniques that will help your audience understand the material. 
+
+## 3. Provide background information and education
+
+Do not assume that the stakeholder knows or does not know certain concepts or terms. However, do assume that they are intelligent and will be able to understand if given an adequate explanation. Do translate key jargon and acronyms that are important to the discussion in a way that makes it simple for the stakeholder to understand. Metaphors and analogies are a good way to translate a technical concept into a more familiar context. It could be useful to give a glossary and definitions of some necessary terms.
```

Comment:
In addition to not assuming the stakeholder knows concepts or terms, it's also wise not to assume they have the full picture or context of why we're having this discussion. So I usually find it beneficial to start off a discussion by quickly recapping where we're at and reminding the audience of the objective of the current discussion. Maybe we can add this point in.

Reply:
This is not really part of translating technical terms for a non-technical audience. I feel this is more on general client/meeting management, which we do not have a chapter currently